#name : try {} catch {}
# --
try {
    $0
} catch (Exception $e) {
    printf('Caught exception : %s', $e->getMessage());
    throw $e;
}
